notebook-images/base/jupyter-minimal-gpu-notebook-imagestream.yaml

```diff
@@ -23,6 +23,8 @@ spec:
       kind: DockerImage
       name: quay.io/opendatahub/workbench-images:cuda-jupyter-minimal-ubi9-python-3.9-2023a-weekly
```

Comment:
very good idea to keep the from.name image reference to the source-repo tag-based on the master branch.